We are building from sdist and so check-manifest always fails.

--- pytest_run_parallel-0.7.0/pyproject.toml.orig
+++ pytest_run_parallel-0.7.0/pyproject.toml
@@ -72,7 +72,6 @@
     "pytest>=6.2.0",
     "coverage[toml]>=7.5",
     "pytest-order",
-    "check-manifest",
     "hypothesis>=6.135.33",
 ]
 commands = [
@@ -92,11 +91,6 @@
     "report",
     "-m"
     ],
-    [
-    "check-manifest",
-    "-u",
-    "-v",
-    ]
 ]
 
 [tool.tox.env.ruff]
